substreams/crates/tycho-substreams/src/entrypoint.rs

```diff
@@ -0,0 +1,49 @@
+use crate::models::{entry_point_params::TraceData, EntryPoint, EntryPointParams};
+
+fn get_entrypoint_id(target: &Vec<u8>, signature: &String) -> String {
```

Comment:
This makes it more flexible

```suggestion
fn get_entrypoint_id(target: &[u8], signature: &str) -> String {
```

Reply:
why?